Formula/s/sigsum.rb

```diff
@@ -0,0 +1,28 @@
+class Sigsum < Formula
```

Comment:
maybe do as sigsum-go, that is what other distributions are doing, https://repology.org/project/sigsum-go/versions. what do you think?

```suggestion
class SigsumGo < Formula
```

Reply:
makes sense, I've renamed it